moreores.lua

```diff
@@ -2,24 +2,9 @@
 
 if bitchange.use_technic_zinc and minetest.get_modpath("technic_worldgen") then
 	minetest.register_craft({
+		type   = "cooking",
 		output = "bitchange:mineninth 8",
```

Comment:
Same here

Reply:
I think that 8 is a pretty small amount already, but I can halve it if you want.